deployments/helm/vpn/templates/vppagent-firewall-nse.tpl

```diff
@@ -15,9 +15,11 @@ spec:
     spec:
       containers:
         - name: firewall-nse
-          image: {{ .Values.registry }}/networkservicemesh/vppagent-firewall-nse:{{ .Values.tag }}
+          image: {{ .Values.registry }}/networkservicemesh/vpp-test-common:{{ .Values.tag }}
           imagePullPolicy: {{ .Values.pullPolicy }}
           env:
+            - name: VPP_APP
```

Comment:
I think we should name this environment variable as TEST_COMMAND instead of VPP_APP.

Reply:
Renamed to `TEST_APPLICATION` : 051dbb4